src/brouwers/kitreviews/models.py

```diff
@@ -21,28 +24,28 @@ class KitReview(models.Model):
     raw_text = models.TextField(
         _(u'review'),
         help_text=_('This is your review. You can use BBCode here.')
-        )
+    )
     html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
-    positive_points = models.TextField(_('positive points'), blank=True)
-    negative_points = models.TextField(_('negative points'), blank=True)
-    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
+    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
+    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
```

Comment:
Abbreviated for clarity, but something like this:

```
class KitReview(models.Model):
    properties = models.ManyToManyField('KitReviewProperty', through='KitReviewPropertyRating')

class KitReviewPropertyRating(models.Model):
    kitreview = models.ForeignKey(...)
    property = models.ForeignKey(...)
    rating = ...
```
 Normally, in a m2m django constructs this exact table transparently, without the `rating` field. This way, you're being explicit about it.

> that would make them required. What if some users are lazy and want only to give general rating and not to rate each prop separately?
This is something that can be handled by form validation. We should have a formset on the page with all the properties 'pre-filled' with the default, half-way (50 out of 100) rating. If they don't touch it then, it will have a neutral rating.

Your point can also be applied to the general rating I'm afraid, if we display a slider (o----|----o), and they don't modify it, you're still stuck with the neutral rating, and we cannot really detect if that's on purpose or not.